src/compute/src/memory_management/memory_manager.rs

```diff
@@ -190,4 +204,10 @@ impl GlobalMemoryManager {
             self.set_watermark_time_ms(watermark_time_ms);
         }
     }
+
+    fn get_eviction_check_interval(params: &SystemParamsReaderRef) -> u64 {
+        // Arbitrarily set a minimal barrier interval in case it is too small,
+        // especially when it's 0.
+        std::cmp::max(params.load().barrier_interval_ms() as u64, 10)
+    }
```

Comment:
May we print a warning somewhere if the user sets a too-small `barrier_interval_ms`?

Reply:
We can check the value range on `ALTER SYSTEM SET`, the question is how small is *too-small* 🤣.